grr/proto/grr_response_proto/osquery.proto

```diff
@@ -14,9 +14,9 @@ enum OsqueryType {
 }
 
 message OsqueryArgs {
+  reserved 3;
```

Comment:
I would not reorder fields and put the `reserved` on the same line for better diffs.